.travis/run-unit-tests.sh

```diff
@@ -20,17 +20,17 @@
 
 install_netdata() {
     echo "Installing Netdata"
-    fakeroot ./netdata-installer.sh --install $HOME --dont-wait --dont-start-it --enable-plugin-nfacct --enable-plugin-freeipmi --disable-lto
+    fakeroot ./netdata-installer.sh --install "$HOME" --dont-wait --dont-start-it --enable-plugin-nfacct --enable-plugin-freeipmi --disable-lto
 }
 
 c_unit_tests() {
     echo "Running C code unit tests"
-    $HOME/netdata/usr/sbin/netdata -W unittest
+    "$HOME/netdata/usr/sbin/netdata" -W unittest
 }
 
 run_c_unit_tests=
 
-if [ -z ${TRAVIS_COMMIT_RANGE} ] ; then
+if [ -z "${TRAVIS_COMMIT_RANGE}" ] ; then
```

Comment:
Can we omit the space preceding `;` as we do in other places?
```suggestion
if [ -z "${TRAVIS_COMMIT_RANGE}" ]; then
```